### PR TITLE
manifest: update fw-nrfconnect-zephyr

### PR DESCRIPTION
**Pull request by rlubos**
Update fw-nrfconnect-zephyr revision to reference latest upmerge.

Signed-off-by: Robert Lubos <robert.lubos@nordicsemi.no>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -34,13 +34,16 @@ manifest:
...
       west-commands: scripts/west-commands.yml
-      revision: de69d2df908fc7bccd8c03b106e7645e90b1503b
+      revision: pull/116/head
     - name: fw-nrfconnect-mcuboot
       path: mcuboot
       revision: 59fde9c792bfaa36887c860fb8cd0ca1f1bc4db5
     - name: nrfxlib
       path: nrfxlib
       revision: 9bcc77b27d12162adc31d8c8c70f4e499338fbdb
+    - name: tinycbor
```
Comment (carlescufi) [inline west.yml]: You will need to specify the remote to be upstream here, we don't have a fork of tinycbor

Comment (carlescufi) [reply west.yml]: I am off today but if in doubt please ask @tejlmand

Comment (rlubos) [reply west.yml]: Perhaps we should set the remote to https://github.com/zephyrproject-rtos/tinycbor? The Zephyr fork is already far behind the upstream.

**Comment by rlubos** [comment]
@SebastianBoe Documentation build complains it cannot find `Kconfig.modules` file:

`kconfiglib.KconfigError: Kconfig.zephyr:10: '/jenkins_workspace/workspace/connect-nrf_documentation_master/_build/Kconfig.modules' not found (in 'source "$(CMAKE_BINARY_DIR)/Kconfig.modules"').`

Any idea on how to fix that? At which point it should be created?

**Pull request merged**